edelphi-persistence/src/main/java/fi/metatavu/edelphi/dao/users/DelfoiUserDAO.java

```diff
@@ -57,19 +57,44 @@ public DelfoiUser findByDelfoiAndUser(Delfoi delfoi, User user) {
   }
 
   public List<DelfoiUser> listByUser(User user) {
```

Comment:
After this change, the method returns all - not just non archived users, so should it be renamed to listAllByUser instead?